packages/dev/core/src/Engines/abstractEngine.ts

```diff
@@ -216,6 +217,9 @@ export abstract class AbstractEngine {
 
     protected _activeRequests: IFileRequest[] = [];
 
+    /** @internal */
+    public _immediateQueue: Array<() => void> = [];

```

Comment:
Is this unused?

Reply:
nice catch! that was my first implementation!